src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -1028,6 +1028,15 @@ void SinglePhaseBase::applySourceFluxBC( real64 const time_n,
       {
         return;
       }
+      if( !subRegion.hasWrapper( dofKey ) )
+      {
+        if( fs.getLogLevel() >= 1 )
+        {
+          GEOS_LOG_RANK( GEOS_FMT( "{}: trying to apply SourceFlux, but set named '{}' intesects with non-simulated region named '{}'.",
```

Comment:
"inteRsects"

Reply:
not sure I understand the "but set named {}" ... Did you try to say: "SourceFlux named '{}' intersects with non-simulated region named '{}'." It will not be applied there."

Reply:
The "set" term here refer to the `targetSet` of the `SourceFlux` (= a `Box` in our case).

Reply:
I can use the term "the targetSet" instead of just "set" to be clearer. The idea is to give as much information as possible.

Reply:
'but its targetSet named {}' ?

Reply:
Ok!